### ISSUE TITLE
As a user I need a user account to store my army lists for future use.

### ISSUE THREAD

**Issue by MaddMatt28** [willy1977/yatac#4]

**Comment by willy1977** [comment]
I was thinking about cloud storage integration for these? e.g. dropbox or box etc. onedrive?

**Comment by MaddMatt28** [comment]
Good idea. Are you planning on providing multiple options, of favouring one over the others?

**Comment by willy1977** [comment]
Thinking I'll setup a separate "enhancement" for each service I'm bothered about, mainly dropbox for me, thinking selfishly, but the aim would be to provide access to major one's.  If we write the system in such a way that others can add on providers without affecting our "base" then the likelihood is support of systems will happen organically

**Comment by MaddMatt28** [comment]
Box for me. Prefer the ui
